addon/templates/components/cf-field/input/table.hbs

```diff
@@ -29,7 +29,12 @@
       </tr>
     </tfoot>
   </table>
-  {{#uk-modal visible=(and showModal documentToEdit) bgClose=false}}
+  {{#uk-modal
+    visible=(and showModal documentToEdit)
+    bgClose=false
+    on-show=(action (mut showModal) true)
```

Comment:
This is not required, right?

Reply:
I removed `on-show`.

I kept back the dependency via commit hash. Either we merge this into the master and update the dependency when we have release the commit or we keep this PR open until ember-uikit is ready.

IMO this PR should not break the master as it will just provide another handler that will not be used.

Reply:
I agree, we can merge this already anyway.